engines/dfc_provider/app/controllers/dfc_provider/enterprise_groups/affiliated_by_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module DfcProvider
+  module EnterpriseGroups
+    class AffiliatedByController < DfcProvider::ApplicationController
+      def create
+        group = EnterpriseGroup.find(params[:enterprise_group_id])
+
+        authorize! :update, group
+
+        enterprise_uri = RDF::URI.new(params[:@id])
+
+        return head :bad_request unless enterprise_uri.valid?
+
+        enterprise_id = ofn_id_from_uri(enterprise_uri)
+        enterprise = Enterprise.find(enterprise_id)
+
+        group.enterprises << enterprise
+
+        return head :unprocessable_entity unless group.save
```

Comment:
The `<<` operator saves the association to the database already. I don't actually know if it would raise or if we would need to check for errors on the record.

Reply:
From what I can see I would expect it to raise an error.